Sources/Tests/UnitTests/Core/Settings.cs

```diff
@@ -73,5 +73,18 @@ public void AssertErrorWhenPoweringNumberAccordingToSetting(string expr)
             Assert.Throws<InvalidArgumentParseException>(() => MathS.FromString(expr));
 
         }
+
+        [Theory]
+        [InlineData("2x")]
+        [InlineData("2(a + x)")]
+        [InlineData("(x+2)(x+5)")]
+        [InlineData("4(8+3)")]
+        public void AssertErrorWhenMultiplyingNumberAccordingToSetting(string expr)
+        {
+            using var _ = ExplicitParsingOnly.Set(true);
+
+            Assert.Throws<InvalidArgumentParseException>(() => MathS.FromString(expr));
+
```

Comment:
Might make sense to remove this line (that's just a spare line, please, don't add them)